pwndbg/emu/emulator.py

```diff
@@ -626,7 +633,8 @@ def hook_intr(self, uc, intno, user_data) -> None:
         """
         We never want to emulate through an interrupt.  Just stop.
         """
-        debug(DEBUG_INTERRUPT, "Got an interrupt")
+        debug(DEBUG_INTERRUPT, "Got an interrupt - %d", intno)
+        self.valid = False
```

Comment:
Previously, an interrupt during a single_step would not cause the step to fail. We set `valid` to False to force it to fail.